server/src/auth-router.ts

```diff
@@ -74,41 +87,34 @@ if (DOMAIN) {
       domain: DOMAIN,
       clientID: CLIENT_ID,
       clientSecret: CLIENT_SECRET,
-      callbackURL:
-        (({
-          stage: 'https://commonvoice.allizom.org',
-          prod: 'https://commonvoice.mozilla.org',
-          dev: 'https://dev.voice.mozit.cloud',
-          sandbox: 'https://sandbox.voice.mozit.cloud',
-        } as any)[ENVIRONMENT] || '') + CALLBACK_URL,
-      scope: 'openid email',
```

Comment:
scope isn't a valid parameter and is never used at this stage, its a param for the `passport.authenticate` method. http://www.passportjs.org/packages/passport-auth0/

I assume this was never working and the default is email, do we let people signup with OpenID?

Reply:
People can sign up using Github, Google, Firefox Accounts, or email, which is split between passwordless (for regular users) and LDAP connected to your Mozilla SSO for Mozilla employees. I would leave scope in here unless you're absolutely certain none of the SSO options require it. 

Also, if you remove the callback URLs I think it'll break login on dev/sandbox.

Reply:
Ah nvm, I see where you added it back in below.